src/PlaywrightSharp.Firefox/FirefoxPage.cs

```diff
@@ -494,7 +504,7 @@ private void OnWorkerCreated(PageWorkerCreatedFirefoxEvent e)
             string workerId = e.WorkerId;
             var worker = new Worker(e.Url);
             FirefoxSession tempWorkerSession = null;
-            var workerSession = new FirefoxSession(_session.Connection, "worker", workerId, async (id, request) =>
+            var workerSession = new FirefoxSession(_session.Connection, "worker", workerId, async (id, request, jsonOptions) =>

```

Comment:
Same, could you explain this?

Reply:
this is in order for us to be able to use custom options that will allow us to handle properties in a special way